operator/pkg/resources/cluster_role.go

```diff
@@ -98,3 +98,26 @@ func (r *ClusterRoleResource) obj() k8sclient.Object {
 func (r *ClusterRoleResource) Key() types.NamespacedName {
 	return types.NamespacedName{Name: "redpanda-init-configurator", Namespace: ""}
 }
+
+// RenderClusterRole generates a cluster role
+func RenderClusterRole(_ context.Context, _ *vectorizedv1alpha1.Cluster) (k8sclient.Object, error) {
+	return &rbacv1.ClusterRole{
+		ObjectMeta: metav1.ObjectMeta{
+			// metav1.ObjectMeta can NOT have namespace set as
+			// ClusterRole is the cluster wide resource.
+			Name:      "redpanda-init-configurator",
+			Namespace: "",
+		},
+		TypeMeta: metav1.TypeMeta{
```

Comment:
There's actually no need to fill out `TypeMeta` here, the Kubernetes client(s) will automatically do so using the `runtime.Scheme` you pass to them.

 We have to do so in the helm chart because gotohelm because they're required for the YAML outputs to helm.